inc/core/templates/theme-notification-banner.php

```diff
@@ -0,0 +1,258 @@
+<?php
+/**
+ * Theme admin functions.
+ *
+ * @package BCGov Block Theme
+ */
+
+namespace Bcgov\Theme\Block;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit( '<h1>Direct access denied.</h1>' );
+}
+
+if ( ! current_user_can( 'manage_options' ) ) {
+	exit( '<h1>You do not have sufficient permissions to edit this page.</h1>' );
+}
+
+$notice_enabled             = esc_attr( get_option( 'notice_enabled' ) );
+$notice_homepage_only       = esc_attr( get_option( 'notice_homepage_only' ) );
+$notice_label_bold          = esc_attr( get_option( 'notice_label_bold' ) );
+$notice_label_big           = esc_attr( get_option( 'notice_label_big' ) );
+$notice_status              = esc_attr( get_option( 'notice_status' ) );
+$notice_content             = esc_attr( get_option( 'notice_content' ) );
+$notice_button_enabled      = esc_attr( get_option( 'notice_button_enabled' ) );
+$notice_button_label        = esc_attr( get_option( 'notice_button_label' ) );
+$notice_button_aria_label   = esc_attr( get_option( 'notice_button_aria_label' ) );
+$notice_button_link         = untrailingslashit( esc_attr( get_option( 'notice_button_link' ) ) );
+$notice_button_utm_campaign = esc_attr( get_option( 'notice_button_utm_campaign' ) );
+
+?>
+
+
+<div class="bcgov-block-theme-page-header">
+	<div class="bcgov-block-theme-page-header__container">
+		<div class="bcgov-block-theme-page-header__branding">
+			<h2><?php esc_html_e( 'Admin Notifications Banner Settings', 'bcgov-block-theme' ); ?> <span class=" <?php echo $notice_enabled ? 'enabled' : 'disabled'; ?> pill">&nbsp;<?php echo $notice_enabled ? 'enabled' : 'disabled'; ?>&nbsp;</span></h2>
+		</div>
+		<div class="bcgov-block-theme-page-header__tagline">
+			<span class="bcgov-block-theme-page-header__tagline-text">
+				<?php esc_html_e( 'Theme repository:', 'bcgov-block-theme' ); ?> <a href="https://github.com/bcgov/bcgov-wordpress-block-theme"><?php echo esc_html__( 'Github', 'bcgov-block-theme' ); ?></a>
+			</span>
+		</div>
+	</div>
+</div><!-- /Header -->
+
+<div class="wrap bcgov-block-theme-container">
+
+	<form method="post" action="options.php">
+
+		<?php
+		settings_fields( 'bcgov-block-theme-notification-banner-group' );
+		do_settings_sections( 'bcgov-block-theme-notification-banner-group' );
+		settings_errors();
+		?>
+
+		<div class="bcgov-block-theme-grid">
+
+			<div class="bcgov-block-theme-grid-content">
+
+				<div class="bcgov-block-theme-body pb-0">
+
+					<h1 class="bcgov-block-theme-title" style="margin-bottom: 1rem;">Notification Banner Settings</h1>
```

Comment:
we should by default wrapping text in `__()' functions, so if the day comes when we have to be completely multi-lang we are one step closer.